gapic-common/lib/gapic/rest/client_stub.rb

```diff
@@ -34,16 +34,22 @@ class ClientStub
       #   (see the [googleauth docs](https://googleapis.dev/ruby/googleauth/latest/index.html))
       # @param numeric_enums [Boolean] Whether to signal the server to JSON-encode enums as ints
       #
+      # @param raise_faraday_errors [Boolean]
+      #   Whether to raise Faraday errors instead of wrapping them in `Gapic::Rest::Error`
+      #   Added for backwards compatibility.
```

Comment:
"Default is true. All REST clients (besides old versions of google-cloud-compute-v1) should explicitly set to false."

Reply:
:+1: